test/regress/cli/regress1/model-blocker-no-spec-values.smt2

```diff
@@ -0,0 +1,31 @@
+; EXPECT: sat
+; EXPECT: unsat
```

Comment:
It depends on what the semantics of block-model means, in this case, it is suggesting that there is only one model that is unique to its criteria.  After this is blocked, then we answer unsat.  This may change if the implementation/semantics of `block-model :values` changes.